frontend/src/views/components/Dashboard/BarChart/BarChart.tsx

```diff
@@ -2,19 +2,57 @@ import { Typography } from "@mui/material";
 import { ResponsiveBar } from "@nivo/bar";
 
 
+const colorMap = {
+    "sender_gender": "#271DE0",
+    "sender_race": "#6820EA",
+    "receiver_gender": "#00D632", 
+    "receiver_race": "#EE55E7",
+};
+
 const BarChart = ({ data }) => (
-        <div style={{height: "300px"}}>
+        <div style={{height: "300px", width: "100%"}}>
             <ResponsiveBar
+                isFocusable={true}
+                barAriaLabel={({ id, value, indexValue }) =>
+                    `The ${indexValue} bar has a score of ${value}`
+                }
+                theme={{
+                    labels: {
+                        text: {
+                            fontSize: '14px',
+                            fontFamily: 'Montserrat',
+                            fontWeight: '400',
+                            fill: '#333',
+                        },
+                    },
+                    axis: {
+                        ticks: {
+                            text: {
+                                fontSize: 12,        // Font size for tick labels
+                                fontFamily: 'Montserrat', // Font family for ticks
+                                fill: '#333',         // Tick color
+                            },
+                        },
+                        legend: {
+                            text: {
+                                fontSize: 14,        // Font size for legends
+                                fontFamily: 'Montserrat', // Font family for legends
+                                fontWeight: 'bold',    // Font weight for legends
+                                fill: '#333',         // Legend color
+                            },
+                        },
+                    },
+                }}
                 data={data}
                 keys={[
                     'score'
                 ]}
                 indexBy="protected_attribute"
```

Comment:
minor nitpick but can we have a larger gap between the bar chart labels and the x axis label